schema/mysql/schema.sql

```diff
@@ -278,6 +278,7 @@ CREATE TABLE servicegroup_customvar (
 
 CREATE TABLE service_state (
   id binary(20) NOT NULL COMMENT 'service.id',
+  host_id binary(20) NOT NULL COMMENT 'service.host.id',
```

Comment:
Just use `host.id` in the comment, tables like `comment`/`downtime`/`notification` also do it that way.